modules/EmailTemplates/EmailTemplate.php

```diff
@@ -70,38 +70,6 @@ class EmailTemplate extends SugarBean
     public $new_schema = true;
     // This is used to retrieve related fields from form posts.
     public $additional_column_fields = array();
-    // add fields here that would not make sense in an email template
-    public $badFields = array(
```

Comment:
We would need to keep this along with the ```generateFieldDefsJS``` and ```generateFieldDefsJS2``` methods if this is going to hotfix-7.10.x, if no longer needed they can be marked as deprecated instead (This is to ensure that any custom code relying on these methods will still work).